spec/support/doc_auth_image_fixtures.rb

```diff
@@ -47,6 +47,22 @@ def self.error_yaml_no_db_multipart
     Rack::Test::UploadedFile.new(path, Mime[:yaml])
   end
 
+  def self.portrait_match_success_yaml
```

Comment:
👍 Interesting, I didn't know about this fixtureing file. I think I just saw some test files that can benefit from this.